tests/snapshot_tests/conftest.py

```diff
@@ -114,14 +124,14 @@ def pytest_sessionfinish(
         actual_svg = item.stash.get(TEXTUAL_ACTUAL_SVG_KEY, None)
         app = item.stash.get(TEXTUAL_APP_KEY, None)
 
-        if snapshot_svg and actual_svg and app:
+        if app:
             path, line_index, name = item.reportinfo()
             diffs.append(
                 SvgSnapshotDiff(
                     snapshot=str(snapshot_svg),
                     actual=str(actual_svg),
                     file_similarity=100
-                    * difflib.SequenceMatcher(
+                                    * difflib.SequenceMatcher(
```

Comment:
Weird formatting? Maybe put the expression in parenthesis. That sometimes prompts black to reformat.

Reply:
Fixed